pkg/prove/proof_test.go

```diff
@@ -87,6 +87,10 @@ func TestTxSharePosition(t *testing.T) {
+		},
```

Comment:
This is my bad for missing this during the review, as it looked like a single line, but why was this added? keeping this in the same line seems kinda insane given the size. If we want to keep it, we should put it in a separate test data file or something

cc @rootulp

Reply:
I'd like to keep this test case b/c it makes a previously sporadic test failure (due to randomness) a concrete test failure.

Agreed, it's a massive line. Will extract to a separate file